src/main/java/duke/task/Event.java

```diff
@@ -0,0 +1,198 @@
+        super(des);
+        this.startTime = startTime;
+        this.endTime = endTime;
+    }
+
+    /**
+     * Return startTime of event.
+     * @return startTime.
+     */
+    public Calendar getStartTime() {
+        return startTime;
+    }
+
+    /**
+     * Return endTime of event.
+     * @return endTime.
+     */
+    public Calendar getEndTime() {
+        return endTime;
+    }
+
+    /**
+     * Returns a string representing startTime in a particular user-friendly format.
+     * @return string representing the startTime in a printing friendly format.
+     */
+    public String printStartTime() {
+        String jargon = "";
+        switch (startTime.get(Calendar.DAY_OF_MONTH) % 10) {
+        case 1 : jargon = "st ";
+            break;
+        case 2 : jargon = "nd ";
+            break;
+        case 3 : jargon = "rd ";
+            break;
+        default : jargon = "th ";
+        }
+        String nameOfMonth = "";
+        switch (startTime.get(Calendar.MONTH)) {
+        case 0 : nameOfMonth = "January";
+            break;
+        case 1 : nameOfMonth = "February";
+            break;
+        case 2 : nameOfMonth = "March";
+            break;
+        case 3 : nameOfMonth = "April";
+            break;
+        case 4 : nameOfMonth = "May";
+            break;
+        case 5 : nameOfMonth = "June";
+            break;
+        case 6 : nameOfMonth = "July";
+            break;
+        case 7 : nameOfMonth = "August";
+            break;
+        case 8 : nameOfMonth = "September";
+            break;
+        case 9 : nameOfMonth = "October";
+            break;
+        case 10 : nameOfMonth = "November";
+            break;
+        case 11 : nameOfMonth = "December";
+            break;
+        default : nameOfMonth = "Invalid Start Month";
+        }
+        String hoursAndMinutes;
+        if (startTime.get(Calendar.HOUR_OF_DAY) <= 12) {
+            hoursAndMinutes = startTime.get(Calendar.HOUR_OF_DAY) + ":";
+        } else {
+            hoursAndMinutes = startTime.get(Calendar.HOUR) + ":";
+        }
+        if (startTime.get(Calendar.MINUTE) < 10) {
+            hoursAndMinutes += "0";
+        }
+        hoursAndMinutes += startTime.get(Calendar.MINUTE);
+        if (startTime.get(Calendar.AM_PM) == 0) {
+            hoursAndMinutes += "am";
+        } else {
+            hoursAndMinutes += "pm";
+        }
+        return startTime.get(Calendar.DAY_OF_MONTH) + jargon
+                + nameOfMonth + ", "
+                + startTime.get(Calendar.YEAR) + ". "
+                + hoursAndMinutes;
+    }
+    
+    /**
+     * Returns a string representing endTime in a particular user-friendly format.
+     * @return string representing the endTime in a printing friendly format.
+     */
+    public String printEndTime() {
+        String jargon = "";
+        switch (endTime.get(Calendar.DAY_OF_MONTH) % 10) {
+        case 1 : jargon = "st ";
```

Comment:
Following the recommended [Java Coding Standard](https://nus-cs2103-ay1920s1.github.io/website/coding-standards/java/intermediate.html) for switch statements, this line should be
`case 1:`
`jargon = "st "`